english/tamilnadu/tamilnadu.md

```diff
@@ -0,0 +1,4 @@
+Tamil Nadu, a southeast Indian state, is famed for its Dravidian-style Hindu temples. 
+In Madurai, Meenakshi Amman Temple has high gopurams (tower gateways) ornamented with colorful figures. 
+Rameswaram temple, on Pamban Island, is a pilgrimage site, and Kanyakumari, at India’s southernmost tip, is the site of ritual sunrises.
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.